cmd/cachecmd/main.go

```diff
@@ -327,11 +341,16 @@ func cacheDir() string {
 
 // REF: https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html
 func xdgCacheHome() string {
-	path := os.Getenv("XDG_CACHE_HOME")
-	if path == "" {
-		path = filepath.Join(os.Getenv("HOME"), ".cache")
+	dir := os.Getenv("XDG_CACHE_HOME")
+	if dir != "" {
+		return
```

Comment:
**[megacheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
wrong number of return values (want 1, got 0)